Pod/Classes/Device.swift

```diff
@@ -56,4 +58,32 @@ public final class Device: Model,ResponseObject,ResponseCollection {
     public func getExperience() -> Experience?{
         return self.experience
     }
+    
+    /**
+     Get Current Device
+     @return Promise<Device>.
+     */
+    public static func getCurrentDevice() -> Promise<Device>{
+        var prom:Promise<Device>?
+        if let identity:[String:AnyObject] = auth?.get("identity") as! [String:AnyObject] {
+            if let uuididentity = identity["uuid"]{
+             prom = Promise { fulfill, reject in
+                Alamofire.request( Router.getDevice(uuididentity as! String) )
+                    .responseObject { (response: Response<Device, NSError>) in
+                        switch response.result{
+                        case .Success(let data):
+                            fulfill(data)
+                        case .Failure(let error):
+                            return reject(error)
+                        }
+                }
+            }
+            }else{
+                prom = Promise(error: NSError(domain: "error", code: 100, userInfo: ["message":"Identity uuid is null"]))
```

Comment:
yeah I will change the docs for this, I'm rejecting the promise with error message I think this is better then return null